spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -359,15 +359,23 @@ public JSONObject confirmLibrariesPoolsUpload(HttpSession session, JSONObject js
                 try {
                   LibraryQC lqc = new LibraryQCImpl();
                   lqc.setLibrary(library);
-                  lqc.setInsertSize(insertSize);
                   lqc.setResults(Double.valueOf(jsonArrayElement.getString(5)));
                   lqc.setQcCreator(user.getLoginName());
                   lqc.setQcDate(new Date());
                   lqc.setQcType(requestManager.getLibraryQcTypeByName("Qubit"));
 
+                  LibraryQC lis = new LibraryQCImpl();
+                  lis.setLibrary(library);
+                  lis.setResults((double) insertSize);
+                  lis.setQcCreator(user.getLoginName());
+                  lis.setQcDate(new Date());
+                  lis.setQcType(requestManager.getLibraryQcTypeByName("Insert Size"));
```

Comment:
I thought the migration created an `Insert Size` QCType for the existing data where we don't know the QC method.